src/Raven.Client/Documents/Session/DocumentQuery.Filter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Raven.Client.Documents.Queries;
+using Raven.Client.Documents.Queries.MoreLikeThis;
+
+namespace Raven.Client.Documents.Session;
+
+public partial class DocumentQuery<T>
+{
+    IDocumentQuery<T> IDocumentQuery<T>.Filter(Action<IFilterFactory<T>> builder)
+    {
+        TurnOnFilter();
```

Comment:
Due to marking this method as `obsolete` we're getting a lot of warnings as you can see